### PR TITLE
Made html changes to allow submit on Enter for the Sign up page

### PR DESCRIPTION
**Pull request by rmiyazaki6499**

### CODE REVIEW

File: frontend/src/app/components/register/register.component.html [hunk 1]
```diff
@@ -1,13 +1,14 @@
-<form (ngSubmit)="processForm()"></form>
+
 <section class="hero is-info">
     <div class="container">
         <div style="text-align: center">
             <h1 class="title">Sign Up!</h1>
-            <input [(ngModel)]="register.username" placeholder="Username" class="input">
+            <form>
+            <input [(ngModel)]="register.username" name="firstname" placeholder="Username" class="input">
```
Comment (alorentsonFS) [inline frontend/src/app/components/register/register.component.html]: `name="firstname"` but all the other attributes refer to `username` is that on purpose?

**Review by alorentsonFS — COMMENTED** [review]
Just curious what the change was that made submit on enter work? I thought it was related to having the form element, but it looks like you removed that?

File: frontend/src/app/components/register/register.component.html (hunk 1 above)
Comment (rmiyazaki6499) [reply frontend/src/app/components/register/register.component.html]: No that's me being stupid

**Comment by rmiyazaki6499** [comment]
I did add the form tag but move it right above the inputs and right under the <h1> title tag.
I saw that's how you did it for the login page so copied that.

**Review by alorentsonFS — APPROVED** [review]

**Comment by alorentsonFS** [comment]
Oh, okay. Yeah I see it now, my bad

**Pull request merged**